pkg/mover/chart.go

```diff
@@ -6,11 +6,17 @@ package mover
 import (
 	"errors"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
+	"path"
+	"path/filepath"
 	"regexp"
 
 	"github.com/avast/retry-go"
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	tarball "github.com/google/go-containerregistry/pkg/v1/tarball"
```

Comment:
This is the reason for all the vendoring changes. Maybe we can separate to another pre-PR?

Reply:
Do you need to rename it?